src/components/home/BlurbHome.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+const BlurbHome = () => {
+  return (
+    <div className="flex w-[30vw] flex-col items-center">
+      <div>
+        <Image
+          src="/home/blurbforhome.jpeg"
+          width={310}
+          height={500}
+          alt="Picture for blurb"
```

Comment:
```suggestion
          alt="Picture for blurb"
```
more specific alt name please